src/Tools/AI Test Toolkit/src/AITTestContextImpl.Codeunit.al

```diff
@@ -270,4 +274,59 @@ codeunit 149043 "AIT Test Context Impl."
 
         CurrentTestOutputJson.Add(ElementName, TestInput.GetTestInput(ElementName).ValueAsText());
     end;
+
```

Comment:
I would place this code under codeunit 149037 "AIT AL Test Suite Mgt"

The Test Context file should be about a test that is currently running, it needs to provide context to it (Answers, questions and etc...)

Reply:
Makes sense, I will move it back.